lib/src/realm_class.dart

```diff
@@ -487,10 +487,29 @@ class Realm implements Finalizable {
     return realmCore.realmEquals(this, other);
   }
 
-  /// The logger to use for logging
-  static Logger logger = Logger.detached('Realm')
-    ..level = RealmLogLevel.info
-    ..onRecord.listen((event) => print(event));
+  static Logger? _logger;
+
+  /// The logger to use for logging.
+  /// The default log level is [RealmLogLevel.info].
+  /// The level could be changed per isolate.
+  /// To manage the log level at runtime use `Realm.logger.level` setter.
```

Comment:
```suggestion
```

Reply:
I think we shouldn't put `logging` package documentation here. Users should be aware of how `Logging` API is used.